falcon/errors.py

```diff
@@ -361,6 +361,25 @@ def __init__(self, title, description, retry_after=None, **kwargs):
                                                   **kwargs)
 
 
+class HTTPUnavailableForLegalReasons(OptionalRepresentation, HTTPError):
+    """451 Unavailable For Legal Reasons.
+
+    This status code indicates that the server is denying access to the
+    resource as a consequence of a legal demand.
+
+    See also:
+    https://datatracker.ietf.org/doc/draft-ietf-httpbis-legally-restricted-status/
+
+    Args:
+        kwargs (optional): Same as for ``HTTPError``.
+
+    """
+
+    def __init__(self, title, **kwargs):
```

Comment:
I noticed that you didn't mention `title` in the docstring. Did you intend for `title` to be required?